src/components/ListPreviousRequests/ListPreviousRequests.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from './ListPreviousRequests.module.css';
+import { Props } from './types';
+
+export default function ListPreviousRequests(props: Props) {
+  const { previousRequests, onClickHandler } = props;
+  if (!previousRequests || previousRequests.length === 0) {
+    return null;
+  }
+  return (
```

Comment:
better `previousRequests && previousRequests.length && ...`
it's not tecessary to return `null` from component